src/SourceBuild/content/eng/pipelines/templates/stages/vmr-scan.yml

```diff
@@ -46,7 +46,7 @@ stages:
         ./.dotnet/dotnet darc vmr scan-cloaked-files
         --vmr "$(Build.SourcesDirectory)"
         --tmp "$(Agent.TempDirectory)"
-        --baseline-file "(Build.SourcesDirectory)/src/installer/src/VirtualMonoRepo/baseline-file.txt"
+        --baseline-file "src/VirtualMonoRepo/baseline-file.txt"
```

Comment:
Also fix the file name to `allowed-binaries.txt`

Reply:
my bad, fixed